packages/create-block/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Master
 
+### New Features
+
+- Added readme.txt file to the existing templates to make your entry in the plugin browser most useful ([#20694](https://github.com/WordPress/gutenberg/pull/20694)).
+- Added prompts for the author, license and version of the plugin ([#20694](https://github.com/WordPress/gutenberg/pull/20694)).
+
```

Comment:
This looks like a changelog for #20694. Should we also have entries for the changes specific to this pull request?

Reply:
I added two new entries in the rebase commit.